### ISSUE TITLE
Documentation: A more comprehensive guide on setting up and hosting.

### ISSUE THREAD

**Issue by vsrisuknimit** [bafolts/terraforming-mars#1205]
Right now the information about how to set up and host are a bit scattered here and there:

- https://github.com/bafolts/terraforming-mars/blob/master/README.md
- https://docs.google.com/document/d/1r4GlqA6DkrSAtR6MMYmX_nmh6o4igVTqDUUETiJYGt8/edit?usp=sharing
- https://docs.google.com/document/d/1y-QnffzkQtpasBkDAFQwBoqhLmUpVTzRPybtvmbktDQ/edit?usp=sharing
- https://drive.google.com/file/d/14hOxxLrCjhWJimvCyuLc-2JRrXevFiR1/view?usp=sharing 
- https://github.com/bafolts/terraforming-mars/wiki/Heroku-Setup

It would be nice to compiled them all nicely into one wiki page with links to more specific instruction page as a sub wiki page. This way we can just give people one link when they ask about how to host.

**Comment by nivaivax** [comment]
Agreed! Also, saw nothing for Linux, so that was a bit of guessing...

Would be good to add some more info on maintenance, eg. what happens if I restart? Best way to access database?
Daemonizing it (eg. pm2)

**Comment by nivaivax** [comment]
Feel free to link on the readme page to the Linux FAQ I made , I think it would be helpful for some, I go back sometimes myself

https://docs.google.com/document/d/1p4u6yHnmlAPHWZUsZoWqPVd6P2427HTWtUeoZKm9OQ8/

**Comment by nivaivax** [comment]
Maybe just a link at the end of the local setup page?
https://github.com/bafolts/terraforming-mars/wiki/Local-Setup
